pom.xml

```diff
@@ -223,5 +223,32 @@ limitations under the License.
         </pluginManagement>
       </build>
     </profile>
+    <profile>
+      <id>release</id>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <artifactId>maven-enforcer-plugin</artifactId>
+              <executions>
+                <execution>
+                  <id>enforce-java-11</id>
```

Comment:
I propose to rename `id` and drop `-11`.
It would
- allow to update l. 242 in the future without the need to update `id` here,
- remove confusion, as currently any >= 11 is allowed, not only 11,
- follow the parent, where `enforce-maven` has no version specified.

Reply:
Done. Thank you for this proposal.